FirebaseAuth/Sources/Swift/SystemService/AuthAPNSTokenManager.swift

```diff
@@ -24,38 +24,39 @@
 
   // Protocol to help with unit tests.
   protocol AuthAPNSTokenApplication {
-    func registerForRemoteNotifications()
+    @MainActor func registerForRemoteNotifications()
   }
 
   extension UIApplication: AuthAPNSTokenApplication {}
 
   /// A class to manage APNs token in memory.
   @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
-  class AuthAPNSTokenManager {
+  class AuthAPNSTokenManager: @unchecked Sendable /* TODO: sendable */ {
     /// The timeout for registering for remote notification.
-    ///
-    /// Only tests should access this property.
-    var timeout: TimeInterval = 5
+    let timeout: TimeInterval
 
     /// Initializes the instance.
     /// - Parameter application: The  `UIApplication` to request the token from.
     /// - Returns: The initialized instance.
```

Comment:
Add `timeout` documentation. Should it continue to be only for tests?

Reply:
Yes. Added back removed docs. Good catch.